src/interactive/repl.ts

```diff
@@ -38,6 +39,65 @@ function startREPLCommand() {
 
     startREPL(false, true)
 }
+async function checkKillPersistentSession() {
+    if (persistentStopWarning || vscode.workspace.getConfiguration('julia').get<boolean>('persistentSession.stopWarning') === false) {
+        return true
+    }
+    else {
+        persistentStopWarning = true
+        try {
+            const agree = 'Yes'
+            const agreeAlways = 'Yes, always'
+            const disagree = 'No'
+            const choice = await vscode.window.showInformationMessage('This is a persistent tmux session. Do you want to close it?', agree, agreeAlways, disagree)
+            if (choice === disagree) {
+                return false
+            }
+            if (choice === agreeAlways) {
+                vscode.workspace.getConfiguration('julia').update('persistentSession.stopWarning', false, true)
+            }
+            if (choice === agree || choice === agreeAlways) {
+                return true
+            }
+        }
+        finally {
+            persistentStopWarning = false
+        }
+    }
+}
+async function stopREPLInPersistent() {
+    const config = vscode.workspace.getConfiguration('julia')
+    if (Boolean(config.get('persistentSession.enabled'))) {
+        try {
+            const sessionName = parseSessionArgs(config.get('persistentSession.tmuxSessionName'))
+            const killSession = await checkKillPersistentSession()
+            if (killSession) {
+                const { code } = await exec(`tmux kill-session -t ${sessionName}`)
+                if (code) {
+                    throw new Error(`tmux kill-session Process failed with exit code ${code}`)
+                }
+            }
+
+        } catch (err) {
+            vscode.window.showErrorMessage('Failed to close tmux session.')
+        }
+    }
+}
+async function stopREPL() {
```

Comment:
This should just be `killREPL` below, no? I don't see the need for an additional method.